gaeun/app.js

```diff
@@ -61,6 +62,36 @@ app.post("/signUp", async (req, res) => {
   res.status(201).json({ message: "userCreated" });
 });
 
+// Bcrypt Verification 및 JWT 발급하기
+app.post("/signIn", async (req, res) => {
+  const { email, password } = req.body;
+
+  const [userData] = await myDataSource.query(
+    `SELECT
+      *
+    FROM 
+      users
+    WHERE 
+      email = ?`,
+    [email]
+  );
+
+  if (!userData) {
+    return res.status(401).json({ message: "Invalid User" });
+  }
+
+  const result = await bcrypt.compare(password, userData.password);
+
+  if (!result) {
+    return res.status(401).json({ message: "Invalid User" });
+  }
+
+  const payLoad = userData.id;
+  const jwtToken = jwt.sign(payLoad, process.env.secretKey);
```

Comment:
```suggestion
  const jwtToken = jwt.sign({ userId: userData.id }, process.env.secretKey);
```
형태가 정확히 맞는진 모르겠으나 이런식으로 작성하면 좋을 것 같습니다.
정확한 네이밍이 필요해보여요~